issue_11/script.js

```diff
@@ -0,0 +1,70 @@
+function startLoading() {
+    const loadingImg = document.createElement('img');
+    loadingImg.src = 'loading-circle.gif';
+    loadingImg.alt = 'ローディング画像';
+    document.getElementById('js-loading').appendChild(loadingImg);
+}
+
+function stopLoading() {
+    document.getElementById('js-loading').remove();
+}
+
+function createLists(resolvedData) {
+    const ul = document.getElementById('js-ul');
+    const fragment = document.createDocumentFragment();
+    for (const item of resolvedData) {
+        const li = document.createElement('li');
+        const a = document.createElement('a');
+        const img = document.createElement('img');
+
+        a.href = item.to;
+        a.textContent = item.text;
+        img.src = item.img;
+        img.alt = item.alt;
+
+        fragment.appendChild(li).appendChild(a).appendChild(img);
+    }
+    ul.appendChild(fragment);
+}
+
+function displayMessage(e) {
+    const ul = document.getElementById('js-ul');
+    ul.textContent = e;
+}
```

Comment:
`e`だとわからないです。ぱっと見eventオブジェクトだと思いました。errorオブジェクトですよね

```suggestion
function displayMessage(e) {
    const ul = document.getElementById('js-ul');
    ul.textContent = e;
}
```

Reply:
I tried to correct it later and overlooked it.
Thank you for pointing this out.
https://github.com/yoshikawa-akr/js-lessons/pull/11/commits/5bd24540591130a6e1634ef98cacdc3658dd7061